tsdb/db_test.go

```diff
@@ -2668,3 +2668,24 @@ func TestChunkWriter(t *testing.T) {
 		})
 	}
 }
+
+func TestRangeForTimestamp(t *testing.T) {
+	type args struct {
+		t     int64
+		width int64
+	}
+	tests := []struct {
+		args     args
+		expected int64
+	}{
+		{args{1, 5}, 5},
```

Comment:
Worth to add some edge cases like `-20` or `0`?  (valid timestamps)

Reply:
like `t=-20, width=5`?  Is there actually a negative value here?